openlibrary/templates/lib/nav_foot.html

```diff
@@ -49,7 +49,7 @@ <h2>Help</h2>
     </div>
     <hr>
     <div id="footer-details">
-      <img id="archive-logo" src="/static/images/pantheon.png">
+      <img id="archive-logo" src="/static/images/pantheon.png" alt="" aria-hidden="true">
```

Comment:
Going from the guidelines here: https://webaim.org/techniques/alttext/ and recommendations here: https://www.w3.org/WAI/tutorials/images/decorative/ I would say that this image is decorative and doesn't add any information to the page, so it would be better with a blank entry.

From the POV of a screenreader the experience would be to skip over the image if were blank, but if we put alt text in we'd get something like "Image, image of the Internet Archive building" being read aloud, which would be misleading.

Reply:
Makes sense, thanks